people.html

```diff
@@ -23,10 +28,24 @@
                 <li class="nav-item">
                     <a class="nav-link" href="wiki.html">Wiki</a>
                 </li>
+                <li class="nav-item">
+                    <i class="fa fa-search search-icon"></i>
+                </li>
             </ul>
         </div>
     </nav>
+
     <main class="container style-fix">
+        <h1>Professor</h1>
+        <div class="container">
```

Comment:
Make this a section tag so we are HTML5 compliant. Whenever possible, we should look to avoid using <div> tags, especially if there is a convenient semantic tag available. [MDN](https://developer.mozilla.org/en-US/docs/Glossary/semantics#semantic_elements)